### PR TITLE
Control reference and property node work

### PR DESCRIPTION
**Pull request by cglzaguilar**

### CODE REVIEW

File: source/io/module_propertyNode.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            MESSAGE: ''
+        },
+
+        kNIObjectReferenceIsInvalid: {
+            CODE: 1055,
+            MESSAGE: 'Object reference is invalid.'
+        }
+    };
+
+    var formatMessageWithException = function (messageText, exception) {
```
Comment (rajsite) [inline source/io/module_propertyNode.js]: This static function is duplicated in module_httpClient and module_javaScriptInvoke.

Please move it to module_coreHelpers and remove the duplicates

File: source/io/module_propertyNode.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        return messageText;
+    };
+
+    // Vireo Core Mixin Function
+    var assignPropertyNode = function (Module, publicAPI) {
+        // Disable new-cap for the cwrap functions so the names can be the same in C and JS
+        /* eslint 'new-cap': ['error', {'capIsNewExceptions': [
+            'NationalInstruments',
+            'DispatchMessagetToHTMLPanel'
+        ]}], */
```
Comment (rajsite) [inline source/io/module_propertyNode.js]: This eslint disable is not needed anymore, delete it all

File: source/io/module_propertyNode.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        Module.propertyNode.jsPropertyNodeWrite = function (
+            viNamePointer,
+            dataItemIdPointer,
+            propertyNamePointer,
+            propertyTypePointer,
+            propertyPathPointer,
+            errorStatusPointer,
+            errorCodePointer,
+            errorSourcePointer) {
+            /* eslint-disable no-undef */
```
Comment (rajsite) [inline source/io/module_propertyNode.js]: Why is this eslint disable here? Remove it or add an elaborate justification in a comment above it

File: source/io/module_propertyNode.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+            var viName = Module.eggShell.dataReadString(viNamePointer);
+            var dataItemId = Module.eggShell.dataReadString(dataItemIdPointer);
+            var propertyName = Module.eggShell.dataReadString(propertyNamePointer);
+            var propertyType = Module.eggShell.dataReadString(propertyTypePointer);
+            var propertyPath = Module.eggShell.dataReadString(propertyPathPointer);
+
+            var context = undefined;
+            try {
+                writeProperty.call(context, viName, dataItemId, propertyName, propertyType, propertyPath);
```
Comment (rajsite) [inline source/io/module_propertyNode.js]: If you want the context to be undefined then just do method invocation

```
writeProperty(viName, dataItemId, propertyName, propertyType, propertyPath);
```

File: source/io/module_propertyNode.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+            var viName = Module.eggShell.dataReadString(viNamePointer);
+            var dataItemId = Module.eggShell.dataReadString(dataItemIdPointer);
+            var propertyName = Module.eggShell.dataReadString(propertyNamePointer);
+            var propertyType = Module.eggShell.dataReadString(propertyTypePointer);
+            var propertyPath = Module.eggShell.dataReadString(propertyPathPointer);
+
+            var context = undefined;
+            try {
+                readProperty.call(context, viName, dataItemId, propertyName, propertyType, propertyPath);
```
Comment (rajsite) [inline source/io/module_propertyNode.js]: same as above, just use function invocation

**Comment by rajsite** [comment]
Waiting on LVJS CI result from @gleonoliva before merging

**Comment by cglzaguilar** [comment]
LVjs CI passed. Rebasing and merging this change.

**Pull request merged**